CHANGELOG.md

```diff
@@ -3,6 +3,17 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] - 2019-08-27
+### Added
+- Support for Docker Auth
+- Ability to store credentials for docker auth in AWS secrets manager
+- Cloudwatch dashboard
+- Cloudwatch alerts
+- SQS queue permissions
+
+### Changed
+- `selected\_tables` variable is now a list 
```

Comment:
Is that `\` supposed to be there? Looks a bit weird in the GitHub markdown view doesn't it? (it shows the `\`)

Reply:
Fixed.